apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts

```diff
@@ -0,0 +1,339 @@
+    public branchList: any;
+    /** Holds Tax Number List */
+    public taxesList: any;
+    /** This will store selected date ranges */
+    public selectedDateRange: any;
+    /** This will store available date ranges */
+    public datePickerOption: any = GIDDH_DATE_RANGE_PICKER_RANGES;
+    /** Selected range label */
+    public selectedRangeLabel: any = "";
+    /** This will store the x/y position of the field to show datepicker under it */
+    public dateFieldPosition: any = { x: 0, y: 0 };
+    /** This will store selected date range to show on UI */
+    public selectedDateRangeUi: any;
+    /** Instance of bootstrap modal */
+    public modalRef: BsModalRef;
+    /** Observable to store the branches of current company */
+    public currentCompanyBranches$: Observable<any>;
+    /** Observable to store the data source of Liability Payment */
+    public liabilityPaymentList$: Observable<any> = this.componentStore.select(state => state.liabilityPaymentList);
+    /** Observable to store the Tax Number */
+    public taxNumber$: Observable<any> = this.componentStore.select(state => state.taxNumber);
+    /** Holds true if multiple branches in the company */
+    public isMultipleBranch: boolean;
+    /** Holds Liabilities Payment Formgroup  */
+    public searchForm: FormGroup;
+    /** Holds table data source */
+    public dataSource: any[] = [];
+    /** Holds Payment table columns */
+    public paymentColumns: string[] = ["index", "received", "amount"];
+    /** Holds Liability table columns */
+    public liabilityColumns: string[] = ["index", "from", "to", "originalAmount", "outstandingAmount", "type", "due"];
+    /** Holds current table columns */
+    public displayColumns: string[] = [];
+    /** Observable to store true if API Call is in progress */
+    public liabilityPaymentListInProgress$ = this.componentStore.select(state => state.liabilityPaymentListInProgress);
+    /** Holds true if user in vat-payment */
+    public isPaymentMode: boolean;
+    /** Stores the current company */
+    public activeCompany: any = {};
+    /** This will hold the value out/in to open/close setting sidebar popup */
+    public asideGstSidebarMenuState: string = 'in';
+    /** True if current company or branch has tax number */
+    public hasTaxNumber: boolean = false;
+    /** Holds current branch information */
+    private currentBranch: any = {};
+
+    constructor(
+        private activatedRoute: ActivatedRoute,
+        private formBuilder: FormBuilder,
+        private store: Store<AppState>,
+        private generalService: GeneralService,
+        private toaster: ToasterService,
+        private modalService: BsModalService,
+        private router: Router,
+        private componentStore: VatReportComponentStore
+    ) {
+        this.initVatLiabilityPaymentForm();
+        this.currentCompanyBranches$ = this.store.pipe(select(appStore => appStore.settings.branches), takeUntil(this.destroyed$));
```

Comment:
this is already available in component store, export from there and use here

Reply:
Done